runtime/src/accounts_db.rs

```diff
@@ -3068,7 +3068,8 @@ impl AccountsDB {
         );
     }
 
-    pub fn compute_merkle_root(hashes: Vec<(Pubkey, Hash, u64)>, fanout: usize) -> Hash {
+    // TODO: remove pub. bench uses it for now.
+    pub fn compute_merkle_root_legacy(hashes: Vec<(Pubkey, Hash, u64)>, fanout: usize) -> Hash {
```

Comment:
don't think we need to keep the legacy version when we have the new one

Reply:
@sakridge and I discussed. I will redo the testing into constants and remove the legacy function in a subsequent pr.